circuit/src/gadgets/mod.rs

```diff
@@ -8,3 +8,13 @@ pub mod lt_eq;
 pub mod main;
 /// Set membership gadget
 pub mod set;
+
+/* NOTE:
+   Following chipsets are not used atm because of column limits.
+   Currently, the maximum columns for EigenTrust circuit is 8 advice + 5 fixed.
+   We plan to use them when lookup table can be used in circuit.
+*/
+// /// Check if a number is less than or equal to a value (using lookup table)
+// pub mod lt_eq_lookup;
+// /// Range check gadget (using lookup table)
+// pub mod range;
```

Comment:
Okay, will update the `range` & `lt_eq_lookup` so that they use kind of mock config.

One question, do you think it is better to leave the `NOTE` comments or remove it after update? @lazovicff

Reply:
I think the NOTE is very useful, so we should leave it there.

Reply:
@duguorong009 Pushed a possible solution.

Reply:
what a trick! :100: @lazovicff 
I think that works best.